src/osd/ECTransaction.h

```diff
@@ -120,6 +132,28 @@ class ECTransaction : public PGBackend::PGTransaction {
     assert(len == bl.length());
     ops.push_back(AppendOp(hoid, off, bl, fadvise_flags));
   }
+  void write(
```

Comment:
As you've started using write in EC backend I suggest to update write method definition at PGTransaction - make it pure virtual and remove out of comment  saying "/// Optional, not supported on ec-pool" ( see PGBackend.h )